vscode-extension/src/extension.ts

```diff
@@ -110,5 +124,18 @@ export async function activate(context: ExtensionContext) {
     );
   });
 
-  context.subscriptions.push(client.start(), restartServerCommand);
+
+  spawn('java', resolveArgs)
+    .on('exit', code => {
+      if (code !== 0) {
+        console.error(
```

Comment:
Do you need to have the console open to see this message? I just tried locally with a dummy serverVersion and couldn't get this message to appear 🤔

Reply:
how about making this a `window.showErrorMessage` instead? I think it's worth being extra explicit when this happens

Reply:
Makes sense, done. I send the errror message to both console & window because I think console should retain history log of important events like this.